resources/kubernetes/operator/helm/values.yaml

```diff
@@ -852,7 +873,7 @@ collectors:
             processors:
               - elasticinframetrics
               - batch
-              - k8sattributes
+              - k8sattributes/ecs
               - resourcedetection/system
```

Comment:
@rogercoll : these changes are not related with the `autoGenerateCert` change I did.
Just FYI.

Reply:
You are right, those changes were already merged. I think this is an issue with how GitHub shows the diffs, see the "two dot" difference instead: https://github.com/elastic/opentelemetry/compare/8.16..main